views/partials/publications/body/view.hbs

```diff
@@ -4,6 +4,13 @@
 
       <h1><span class="text-accent">{{publication.type}}:</span> {{publication.title}}</h1>
 
+      <form action="/publications/download-pdf/{{publication._id}}" method="GET">
```

Comment:
Any reason for this to be a form rather than a simple, direct link?

Reply:
Good catch. Wrapping the button in a simple anchor tag should have the same behavior 👍